frontend/src/pages/Visualizer/utils/mock.tsx

```diff
@@ -0,0 +1,1918 @@
+const mockStructure = {
```

Comment:
You should use JSON to pass the mocked data, and use `require('xxx..json')` to load the JSON file. Data should not be kept in typescript.